tests/benchmarks/funcx_tasks/main_test.py

```diff
@@ -35,7 +35,8 @@ def test_time_task() -> None:
 
 def test_time_task_proxy() -> None:
     fx = mock_executor()
-    store = init_store('local', 'test-store', stats=True)
+    store = LocalStore(name='test-time-task-store', stats=True)
+    register_store(store)
```

Comment:
This should've been in the original code, but anywhere you call `register_store` in a test, can you add a corresponding `unregister_store(store.name)` at the end of the test?